.gitignore

```diff
@@ -0,0 +1,2 @@
+*~
+.DS_Store
```

Comment:
IMHO, I think these better go to your [global gitignore file](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore).